packages/cli/src/commands/generate/function/templates/function.ts.template

```diff
@@ -0,0 +1,10 @@
+import { APIGatewayEvent, Context } from 'aws-lambda'
+
+export const handler = async (event: APIGatewayEvent, context: Context) => {
```

Comment:
I would love to add some syntax to help our none-typescript users out.

```suggestion
/**
 * The handler function is your code that processes http request events.
 * You can use return and throw to send a response or error, respectively.
 * 
 * @param { import("'aws-lambda'").APIGatewayEvent } event - an object which contains information from the invoker.
 * @param { import("'aws-lambda'"). Context } context - contains information about the invocation,
 * function, and execution environment.
 */
export const handler = async (event: APIGatewayEvent, context: Context) => {
```

Also, does AWS lambda has a return type?

Reply:
I'm not 100% sure that that syntax will work. If it doesn't then we'll have to use @typedef.

Reply:
@peterp Why the double quotes in the import?

This is how you'd import with typedef
```
/**
 * @typedef {import('./Foo').default} Bar
 */
 ```

Reply:
Ah, thanks for catching that... turns out textareas aren't great code editors.

Reply:
Didn't get a notification for this sorry. [Netlify's documentation](https://docs.netlify.com/functions/build-with-javascript/#synchronous-function-format) is where I got direction for the parameter types. [This article](https://serverlessfirst.com/fff-aws-lambda-type-definitions/) provides a return type from the same package, haven't tested that as I just found it. I've never used lambda so I don't know what it's expecting as a result.